client/src/components/map/legend/item.tsx

```diff
@@ -7,7 +7,7 @@ export interface LegendItemProps {
   type: "basic" | "gradient" | "choropleth";
   items: {
     id: string | number;
-    label: string | null;
+    label?: string;
```

Comment:
The label can be null, don't change the typing because of the translations. There are cases where we have 5 items but only 2 labels, the first and the last one.